cargo-dist/tests/integration-tests.rs

```diff
@@ -747,6 +747,7 @@ targets = ["x86_64-unknown-linux-gnu", "aarch64-unknown-linux-gnu", "x86_64-unkn
 ci = ["github"]
 
 [workspace.metadata.dist.github-custom-runners]
+global = "buildjet-8vcpu-ubuntu-2204"
```

Comment:
so "global" is the obvious term for this but i wonder if it's a bit too inside baseball to expose to end users? i am ok with keeping it for now, my brain kinda goes to `"default"` e.g. use this runner when the runner doesn't really matter, but im' not sure if that's actually better or not

Reply:
also: do we want to run our tests on the private runner? it's not a lot of money, but i feel like it might be good to not do that if we don't have to

Reply:
Happy to defer to your opinions on the public-facing name for this.

This test doesn't actually run in CI; I'm just testing with this that it generates the expected CI config.

Reply:
i think we should probably ship with global for now, but when we do the config 1.0 project consider the way we talk about jobs in general!